lldb/source/API/SBTarget.cpp

```diff
@@ -1326,23 +1326,35 @@ lldb::SBWatchpoint SBTarget::WatchAddress(lldb::addr_t addr, size_t size,
                                           SBError &error) {
   LLDB_INSTRUMENT_VA(this, addr, size, read, write, error);
 
+  SBWatchpointOptions options;
+  options.SetWatchpointTypeRead(read);
+  options.SetWatchpointTypeModify(write);
```

Comment:
do we want to rename this argument from "write" to "modify" to clarify what will happen? Variable names are not included in the mangled name, so we can rename as long as we don't change the type and not affect the API backwards compat. If we do rename, then update the header file as well

Reply:
Good point.